app/helpers/notifications_helper.rb

```diff
@@ -0,0 +1,8 @@
+module NotificationsHelper
+  def create_system_notification(title, message)
+    users = User.where.not(confirmed_at: nil)
+    users.each do |u|
+      UserNotification.create_notification(u, title, message, :system_message)
```

Comment:
Wasn't one of the purposes of `Notification` - `UserNotification` - `User` database structure that for system notification, a **single** `Notification` object would be created, and linked to many `UserNotification`-s? Because right now, for each system notification, many `Notification` objects are created, and I think that's very redundant.